packages/ketcher-core/src/application/render/restruct/rebond.ts

```diff
@@ -59,8 +60,18 @@ class ReBond extends ReObject {
 
   static bondRecalc(bond: ReBond, restruct: ReStruct, options: any): void {
     const render = restruct.render;
-    const atom1 = restruct.atoms.get(bond.b.begin);
-    const atom2 = restruct.atoms.get(bond.b.end);
+    const sgroup1 = restruct.molecule.getGroupFromAtomId(bond.b.begin);
+    const sgroup2 = restruct.molecule.getGroupFromAtomId(bond.b.end);
+    const atom1 = restruct.atoms.get(
```

Comment:
Could we rename the atom1 to atomStart maybe? alos the others

Reply:
done